dubbo-rpc/dubbo-rpc-rest/src/main/java/org/apache/dubbo/rpc/protocol/rest/RestProtocol.java

```diff
@@ -136,65 +133,77 @@ protected <T> Runnable doExport(T impl, Class<T> type, URL url) throws RpcExcept
 
     @Override
     protected <T> T doRefer(Class<T> serviceType, URL url) throws RpcException {
+        ReferenceCountedClient referenceCountedClient = clients.computeIfAbsent(url.getAddress(), _key -> {
```

Comment:
if the ip:port is the same,but httpclient config properties are different, how to fix th is situation?